### DISCUSSION TITLE
Facebook снова не работает

### DISCUSSION THREAD

**Discussion by Olezha180** [anticensority/runet-censorship-bypass#347] [Q&A]
Facebook снова не работает

**Comment by ilyaigpetrov** [comment]
Пришлите снимок проблемы, с которой вы сталкиваетесь (ошибка с кодом, поломка интерфейса и т.д.).
У меня в данный момент через АнтиЗапрет fb работает.

**Reply by Olezha180** [reply]
> Пришлите снимок проблемы, с которой вы сталкиваетесь (ошибка с кодом, поломка интерфейса и т.д.). У меня в данный момент через АнтиЗапрет fb работает.

Сегодня опять не работает

**Comment by Olezha180** [comment]
----------------Кому: anticensority/runet-censorship-bypass ***@***.***);Копия: Author ***@***.***);Тема: [anticensority/runet-censorship-bypass] Facebook снова не работает (Discussion #347);24.12.2024, 19:47, "ilyaigpetrov" ***@***.***>: Пришлите снимок проблемы, с которой вы сталкиваетесь (ошибка с кодом, поломка интерфейса и т.д.).У меня в данный момент через АнтиЗапрет fb работает.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>  -- С уважением,Ситников Олег+79226170422

**Reply by ilyaigpetrov** [reply]
Олег, что-то не то вы прислали. Пришлите снимок экрана, где видна проблема, с которой вы сталкиваетесь.

**Comment by Olezha180** [comment]
Три точки наверху видете? Дальше что? Куда ткнуть? ----------------Кому: anticensority/runet-censorship-bypass ***@***.***);Копия: Author ***@***.***);Тема: [anticensority/runet-censorship-bypass] Facebook снова не работает (Discussion #347);24.12.2024, 19:53, "ilyaigpetrov" ***@***.***>: Олег, что-то не то вы прислали. Пришлите снимок экрана, где видна проблема, с которой вы сталкиваетесь.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>  -- С уважением,Ситников Олег+79226170422

**Reply by ilyaigpetrov** [reply]
Три точки не вижу. Снимка тоже не получал.

**Comment by Olezha180** [comment]
Почему у меня торренты и вебкам-сайты открываются, а ФБ - нет? ----------------Кому: anticensority/runet-censorship-bypass ***@***.***);Копия: Author ***@***.***);Тема: [anticensority/runet-censorship-bypass] Facebook снова не работает (Discussion #347);24.12.2024, 20:16, "ilyaigpetrov" ***@***.***>: Три точки не вижу. Снимка тоже не получал.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>  -- С уважением,Ситников Олег+79226170422

**Reply by ilyaigpetrov** [reply]
Для FB, Twitter/X.com, instagram используются отдельные прокси. Так что торрент-трекеры и FB работаеют через разные сервера.

**Comment by Olezha180** [comment]
Получили две фотки сейчас? ----------------Кому: anticensority/runet-censorship-bypass ***@***.***);Копия: Author ***@***.***);Тема: [anticensority/runet-censorship-bypass] Facebook снова не работает (Discussion #347);24.12.2024, 20:22, "ilyaigpetrov" ***@***.***>: Для FB, Twitter/X.com, instagram используются отдельные прокси. Так что торрент-трекеры и FB работаеют через разные сервера.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>  -- С уважением,Ситников Олег+79226170422

**Reply by ilyaigpetrov** [reply]
Нет, не получил. Проверил почту, данное обсуждение на github и telegram -- нигде нет.

**Comment by Olezha180** [comment]
Напишите свою почту, пожалуйста ----------------Кому: anticensority/runet-censorship-bypass ***@***.***);Копия: Author ***@***.***);Тема: [anticensority/runet-censorship-bypass] Facebook снова не работает (Discussion #347);24.12.2024, 20:26, "ilyaigpetrov" ***@***.***>: Нет, не получил. Проверил почту, данное обсуждение на github и telegram -- нигде нет.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>  -- С уважением,Ситников Олег+79226170422

**Reply by ilyaigpetrov** [reply]
Я не хочу раскрывать свою почту в публичной переписке.
Вы можете прикрепить к ответу снимки через эту кнопку:

![gh-attach-images](https://github.com/user-attachments/assets/c00fb7ff-1c42-4fc0-9384-b1ffe5a1dfb8)

**Reply by Olezha180** [reply]
> Я не хочу раскрывать свою почту в публичной переписке. Вы можете прикрепить к ответу снимки через эту кнопку:
> 
> ![gh-attach-images](https://private-user-images.githubusercontent.com/1090811/398440971-c00fb7ff-1c42-4fc0-9384-b1ffe5a1dfb8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwNTU1NjcsIm5iZiI6MTczNTA1NTI2NywicGF0aCI6Ii8xMDkwODExLzM5ODQ0MDk3MS1jMDBmYjdmZi0xYzQyLTRmYzAtOTM4NC1iMWZmZTVhMWRmYjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMjRUMTU0NzQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmM1NmZkYzhkMjJhZWM0MjUyNWFmNzkzMmUwOTU3MzQ2ZTRlYjA3ZDQ1NmU3ZWY5NmZkOTJhOTRkNDY5NzFmYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.E0J_zaWiBV9v4TFinn0sC15YdyG8orZcYaYIis1Wxuc)

![20241224_200706](https://github.com/user-attachments/assets/c698e91d-b9ef-47a3-8d91-0e7832022ed3)
![20241224_200717](https://github.com/user-attachments/assets/05013943-862a-4246-8c5e-fcd5a58df987)

**Reply by Olezha180** [reply]
> Я не хочу раскрывать свою почту в публичной переписке. Вы можете прикрепить к ответу снимки через эту кнопку:
> 
> ![gh-attach-images](https://private-user-images.githubusercontent.com/1090811/398440971-c00fb7ff-1c42-4fc0-9384-b1ffe5a1dfb8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUxNDQ4MDIsIm5iZiI6MTczNTE0NDUwMiwicGF0aCI6Ii8xMDkwODExLzM5ODQ0MDk3MS1jMDBmYjdmZi0xYzQyLTRmYzAtOTM4NC1iMWZmZTVhMWRmYjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIyNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMjVUMTYzNTAyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDIyZjJhZTJmOTFjNjI2YTIzM2U3YWVlMjM4MWM1NWNkMzg0N2Q5NTA1ZjcyZTVmZDA3N2Y4MTM4MGM2ODQ5NSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.BcLlLiUqxUWr5pyr0HZMhiU7srVxsyn-r0pvK6UF4qw)

Сегодня опять какие-то проблемы
